src/components/List/BaseList.tsx

```diff
@@ -69,20 +89,30 @@ const BaseList = ({
                 <List
                     itemLayout="horizontal"
                     header={
-                        <>
+                        <Row style={{width: "100%"}}>
                             <Row
                                 style={{
                                     fontSize: 18,
+                                    width: "100%"
                                 }}
                             >
                                 <span>{title}</span>
                             </Row>
-                            <Row>
+                            <Row
+                                style={{
+                                    width: "100%"
+                                }}
+                            >
+                                <Col>
                                 {t("list:totalItems", {
                                     total: totalItems
                                 })}
+                                </Col>
+                                <Col>
```

Comment:
I think you still need to make the style part of the component to match the figma design. Let me know if you need any help with that.